### PR TITLE
Rename bin/webpack[-dev-server] to bin/webpacker[-dev-server]

### PR DESCRIPTION
**Pull request by justin808**
Renamed `/bin/webpack` to `/bin/webpacker` and `/bin/webpack-dev-server`
to `bin/webpacker-dev-server` to avoid confusion with underlying webpack
executables.

Addresses #3237.

### CODE REVIEW

**Comment by justin808** [comment]
CI passes. What else should be tested? Most changes are in the docs.

File: docs/v6_upgrade.md [hunk 1]
```diff
@@ -103,14 +103,16 @@ Example going to a specific version:
...
-17. Make sure that you can run `bin/webpack` without errors.
+17. Make sure that you can run `bin/webpacker` without errors.
 
 18. Try running `RAILS_ENV=production bin/rails assets:precompile`. If all goes well, don't forget to clean the generated assets with `bin/rails assets:clobber`.
 
 19. Run `yarn add webpack-dev-server` if those are not already in your dev dependencies. Make sure you're using v4+.
 
 20. Try your app!
 
+21. Update any scripts that called `/bin/webpack` or `bin/webpack-dev-server` to `/bin/webpacker` or `bin/webpacker-dev-server`
```
Comment (nguyenthetoan) [inline docs/v6_upgrade.md:114]: I think this should be the 20th step, the last step should always be `Try your app!` 😀

Comment (frullah) [inline docs/v6_upgrade.md:114]:
```suggestion
21. Update any scripts that called `bin/webpack` to `bin/webpacker` and `bin/webpack-dev-server` to `bin/webpacker-dev-server`
```

Comment (justin808) [reply docs/v6_upgrade.md:114]: We've got some other PRs with changes here, so I'll clean up all at once!

**Comment by guillaumebriday** [comment]
Nice!

**Pull request merged**